orchestrator-salt/src/main/resources/salt-common/salt/fluent/template/output.conf.j2

```diff
@@ -56,7 +56,11 @@
     azure_storage_account    {{fluent.azureStorageAccount}}
     azure_container          {{fluent.azureContainer}}
 {% if fluent.azureInstanceMsi is defined and fluent.azureInstanceMsi is not none and fluent.azureInstanceMsi %}
+    {% if grains['hostgroup'] == "idbroker" %}
+    azure_instance_msi       {{fluent.azureIdBrokerInstanceMsi}}
```

Comment:
At first sight this did not make sense to me as log shipping normally does not work on the IDBroker node (see [CB-5107](https://jira.cloudera.com/browse/CB-5107)). I guess this setting only works in case of internal E2E testing when the IDBroker VM MSI (i.e. the assumer identity) also contains the necessary role assignment for accessing the log storage account location. And similarly for AWS, the IDBroker EC2 instance profile role should have a policy for accessing the log S3 bucket.

Reply:
just to answer that here as well, it's better now to fail during log shipping (because of the lack of proper contributor roles), then fail during obtaining the azure access token (because of wrong vm identity). some would happen for AWS if idbroker instance profile does not have the right policies